src/styles/variables.scss

```
@@ -238,29 +297,41 @@ $button-default-disabled-color: var(
+) !default;
 $button-default-font-size: var(
   --nutui-button-default-font-size,
   $font-size-base
 ) !default;
 
-$button-xlarge-height: var(--nutui-button-xlarge-height, 48px) !default;
-$button-xlarge-padding: var(--nutui-button-xlarge-padding, 0px 24px) !default;
+$button-xlarge-height: var(
+  --nutui-button-xlarge-height,
+  scale-px(48px)
+) !default;
```

Stylelint (17.11.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)